sources/api/netdog/src/networkd/config/mod.rs

```diff
@@ -23,3 +23,67 @@ impl NetworkDConfigFile {
         }
     }
 }
+
+// This private module defines some empty traits meant to be used as type parameters for the
```

Comment:
nit: doc comments seem fine to me everywhere they are allowed, even on private symbols. They have the benefit of rendering some markdown syntax highlighting.